fedimint-server/src/consensus/engine.rs

```diff
@@ -650,15 +654,15 @@ impl ConsensusEngine {
         // When we recover from a mid-session crash aleph bft will replay the units that
         // were already processed before the crash. We therefore skip all consensus
         // items until we have seen every previously accepted items again.
-        if let Some(accepted_item) = dbtx
```

Comment:
Can revert. I think 'existing' carries the meaning better here, at least from a pov of outside investigator.